src/clusterfuzz/_internal/bot/tasks/commands.py

```diff
@@ -80,6 +80,8 @@ class AlreadyRunningError(Error):
 def cleanup_task_state():
   """Cleans state before and after a task is executed."""
   # Cleanup stale processes.
+  if not environment.is_tworker():
```

Comment:
How bad is it if this codepath gets hit by the uworker? Why not an assertion?

Reply:
I expect the codepath to be hit by both uworker and tworker (is that what you mean).
I just don't want to do the cleanup in a twoker. It's pretty slow and it's doing things a tworker doesn't need like killing processes and removing tmpdirs. tworker doesn't really need this because it's not actually running or downloading anything.